src/test/java/seedu/algobase/logic/commands/DeleteTagCommandTest.java

```diff
@@ -0,0 +1,13 @@
+package seedu.algobase.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.algobase.logic.commands.DeleteTagCommand.COMMAND_WORD;
+
+import org.junit.jupiter.api.Test;
+
+public class DeleteTagCommandTest {
+    @Test
+    public void testcommandword() {
```

Comment:
Naming of your tests are not consistent with AddressBook, you may refactor them in v1.3.